add dsn to unity getting started (#3546)
getsentry/sentry-docs: src/includes/getting-started-config/unity.mdx

Currently, you can only configure Sentry using the Unity tools window.

> The minimum required configuration, is to set your DSN in the window. The DSN is available in your Sentry project settings, or if you're logged in, you can copy from below:

Reviewer: ```suggestion
The minimum required configuration is to set your [DSN](/product/sentry-basics/dsn-explainer/) in the window. The DSN is available either in your Sentry project settings or, if you're logged in, by copying:
```

```
___PUBLIC_DSN___
```

> To configure it, open `Tools` -> `Sentry`.

Reviewer: ```suggestion
To configure the DSN, open `Tools` -> `Sentry`.
```

![Sentry tools](tools_sentry.png)

And the Sentry configuration window will open:

![Sentry window](sentry_window.png)

> After you close the window, all settings are saved into `Assets/Resources/Sentry/SentryOptions.json`. This file should be saved in your version control system.

Reviewer: ```suggestion
After you close the window, all settings are saved to `Assets/Resources/Sentry/SentryOptions.json`, which should be maintained in your version control system.
```
